samples/wifi/radio_test/src/nrf_wifi_radio_test_shell.c

```diff
@@ -319,7 +327,17 @@ enum wifi_nrf_status nrf_wifi_radio_test_conf_init(struct rpu_conf_params *conf_
 	conf_params->ru_index = 1;
 	conf_params->tx_pkt_cw = 15;
 	conf_params->phy_calib = NRF_WIFI_DEF_PHY_CALIB;
-	memcpy(conf_params->country_code, "00", 3);
+
+	/* Store back the currently set country code */
+	if (strlen(country_code)) {
+		memcpy(conf_params->country_code,
+		       country_code,
+		       NRF_WIFI_COUNTRY_CODE_LEN);
+	} else {
```

Comment:
But we are printing character, Initializig the memory to zero, will not print as expected. Mostly It will be required. Earlier also memcpy was there. "memcpy(conf_params->country_code, "00", 3)";  Will test and update.